MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -1715,26 +1716,52 @@ function access_ADDR_panel(access_type, notes_on_address, resi_line_one, resi_li
 	        Call create_mainframe_friendly_date(addr_eff_date, 4, 43, "YY")
 			
 			'This functionality will write the address word by word so that if it needs to word wrap to the second line, it can move the words to the next line
-			If len(resi_street_full) > 22 Then 
-	            resi_words = split(resi_street_full, " ")
-	            write_resi_line_one = ""
-	            write_resi_line_two = ""
-	            For each word in resi_words
-	                If write_resi_line_one = "" Then
-	                    write_resi_line_one = word
-	                ElseIf len(write_resi_line_one & " " & word) =< 22 Then
-	                    write_resi_line_one = write_resi_line_one & " " & word
-	                Else
-	                    If write_resi_line_two = "" Then
-	                        write_resi_line_two = word
-	                    Else
-	                        write_resi_line_two = write_resi_line_two & " " & word
-	                    End If
-	                End If
-	            Next
-	        Else
-				write_resi_line_one = resi_street_full
-	        End If
+			If resi_line_one = "" AND resi_line_two = "" Then 
+				If len(resi_street_full) > 22 Then 
+					resi_words = split(resi_street_full, " ")
+					write_resi_line_one = ""
+					write_resi_line_two = ""
+					For each word in resi_words
+						If write_resi_line_one = "" Then
+							write_resi_line_one = word
+						ElseIf len(write_resi_line_one & " " & word) =< 22 Then
```

Comment:
If a smaller word is after a longer word - the order or words will get messed up. 

'1135 Fifth Avenue Apartment 1905' became 

'1135 Fifth Avenue 1905'
'Apartment'

We need a boolean to identify if we are moving to line 2

Reply:
@C-Love - I've added boolean for resi and mail handling. Take a peek and let me know what you think!